tests/test_Tensor_view.py

```diff
@@ -0,0 +1,234 @@
+        result = a.view((k, k))
+        """
+    )
+    obj.run(pytorch_code, ["result"])
+
+
+def test_case_7():
+    pytorch_code = textwrap.dedent(
+        """
+        import torch
+        a = torch.arange(2.)
+        k = 2
+        result = a.view(k)
+        """
+    )
+    obj.run(pytorch_code, ["result"])
+
+
+# # Because the current paddle.view does not support tensors without a shape, this case cannot run properly.
+# def test_case_9():
+#     pytorch_code = textwrap.dedent(
+#         """
+#         import torch
+#         a = torch.tensor(1.)
+#         result = a.view(1)
+#         """
+#     )
+#     obj.run(pytorch_code, ["result"])
+
+
+def test_case_8():
+    pytorch_code = textwrap.dedent(
+        """
+        import torch
+        a = torch.arange(6.)
+        result = a.view((2, 3))
+        """
+    )
+    obj.run(pytorch_code, ["result"])
+
+
+def test_case_9():
+    pytorch_code = textwrap.dedent(
+        """
+        import torch
+        a = torch.arange(6.)
+        result = a.view(torch.int32)
+        """
+    )
+    obj.run(pytorch_code, ["result"])
+
+
+def test_case_10():
+    pytorch_code = textwrap.dedent(
+        """
+        import torch
+        a = torch.arange(6.)
+        k = torch.int32
+        result = a.view(k)
+        """
+    )
+    obj.run(pytorch_code, ["result"])
+
+
+def test_case_11():
+    pytorch_code = textwrap.dedent(
+        """
+        import torch
+        a = torch.arange(6)
+        result = a.view(dtype = torch.int32)
+        """
+    )
+    obj.run(pytorch_code, ["result"])
+
+
+def test_case_12():
+    pytorch_code = textwrap.dedent(
+        """
+        import torch
+        a = torch.arange(6)
+        result = a.view(dtype = torch.float32)
+        """
+    )
+    obj.run(pytorch_code, ["result"])
+
+
+def test_case_13():
+    pytorch_code = textwrap.dedent(
+        """
+        import torch
+        a = torch.arange(6)
+        result = a.view(dtype = torch.float)
+        """
+    )
+    obj.run(pytorch_code, ["result"])
+
+
+def test_case_14():
+    pytorch_code = textwrap.dedent(
+        """
+        import torch
+        a = torch.arange(6)
+        result = a.view(dtype = torch.float64)
+        """
+    )
+    obj.run(pytorch_code, ["result"])
+
+
+def test_case_15():
+    pytorch_code = textwrap.dedent(
+        """
+        import torch
+        a = torch.arange(6)
+        result = a.view(dtype = torch.double)
+        """
+    )
+    obj.run(pytorch_code, ["result"])
+
+
+def test_case_16():
+    pytorch_code = textwrap.dedent(
+        """
+        import torch
+        a = torch.arange(6)
+        result = a.view(dtype = torch.cfloat)
+        """
+    )
+    obj.run(pytorch_code, ["result"])
+
+
+def test_case_17():
+    pytorch_code = textwrap.dedent(
+        """
+        import torch
+        a = torch.arange(6)
+        result = a.view(dtype = torch.half)
+        """
+    )
+    obj.run(pytorch_code, ["result"])
+
+
+def test_case_18():
```

Comment:
指定关键字不能是shape吗？`torch.Tensor.view` 的单测需要再梳理合并下，增加case种类，而不是case个数，减少重复case

Reply:
是的，指定关键字不能是 shape, 示例代码如下：

```python
imort torch
x = torch.arange(4)

x.view(shape=(2,2))
# 报错信息如下：
# TypeError: view() received an invalid combination of arguments - got (shape=tuple, ), but expected one of:
# * (tuple of ints size)
#      didn't match because some of the keywords were incorrect: shape
# * (torch.dtype dtype)
 #     didn't match because some of the keywords were incorrect: shape
```

针对单测中，涉及的多个 dtype 类型是因为，torch.view 在改变张量类型时，当 dtype 大于或小于 self.dtype 的时候会对张量进行改变，如下代码所示：

```python
imort torch
x = torch.arange(4)

print(x.view(torch.bool))
# tensor([False, False, False, False, False, False, False, False,  True, False,
#        False, False, False, False, False, False,  True, False, False, False,
#       False, False, False, False,  True, False, False, False, False, False,
#       False, False])

print(x.view(torch.half))
# tensor([0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.9605e-08, 0.0000e+00,
#        0.0000e+00, 0.0000e+00, 1.1921e-07, 0.0000e+00, 0.0000e+00, 0.0000e+00,
#        1.7881e-07, 0.0000e+00, 0.0000e+00, 0.0000e+00], dtype=torch.float16)

print(x.view(torch.cfloat))
# tensor([0.0000e+00+0.j, 1.4013e-45+0.j, 2.8026e-45+0.j, 4.2039e-45+0.j])

print(x.view(torch.uint8))
# tensor([0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0,
#        3, 0, 0, 0, 0, 0, 0, 0], dtype=torch.uint8)
```

Reply:
好的，dtype确实需要多写几种。还有一种关键字用法 `x.view(size=[2, 2])`

Reply:
好的，在下一个 commit 中补充该测试案例。